environment-pipelines/buildspec-plan.yml

```diff
@@ -45,7 +45,7 @@ phases:
             platform-helper notify add-comment "${SLACK_CHANNEL_ID}" "${SLACK_TOKEN}" "${SLACK_REF}" "Waiting for approval for the ${ENVIRONMENT} environment."
           fi
         else
-          platform-helper notify add-comment "${SLACK_CHANNEL_ID}" "${SLACK_TOKEN}" "${SLACK_REF}" "Terraform plan phase FAILED for the ${ENVIRONMENT} environment."
+          platform-helper notify add-comment "${SLACK_CHANNEL_ID}" "${SLACK_TOKEN}" "${SLACK_REF}" ":alert: @here Terraform plan phase FAILED for the ${ENVIRONMENT} environment."
```

Comment:
I think this would need to be a standalone message, not a thread reply, as @here doesn't work in threads.

Reply:
Seemed to work in your tests earlier though, so lets see how it plays out in the wild.